core/domain/rights_manager.py

```diff
@@ -247,7 +311,7 @@ def get_multiple_exploration_rights_by_ids(exp_ids):
     """
     exp_rights_models = exp_models.ExplorationRightsModel.get_multi(
         exp_ids)
-    exp_models_list = []
+    exp_models_list: List[Optional[rights_domain.ActivityRights]] = []
```

Comment:
This seems mistyped or misnamed. Shouldn't a variable called exp_models_list contain exploration models rather than rights models?

Reply:
naming changed.